### PR TITLE
reading update

### PR DESCRIPTION
**Pull request by AitorBengoechea**

### CODE REVIEW

File: sandy/groupr.py [hunk 1]
```diff
@@ -147,11 +166,13 @@ def get_xs(self, **kwargs):
                                           listmt=listmt_,
                                           listmat=listmat_).data:
             mf3 = sandy.groupr.read_mf3(self, mat, mt)
-            xs = np.array([x["DATA"][1].tolist() for x in mf3["GROUPS"]]).T
+            lowest_range = mf3["GROUPS"][0]["IG"] - 1
+            xs = np.array([x["DATA"][1].tolist() for x in mf3["GROUPS"]])
+            xs = np.insert(xs, [0]*lowest_range, 0) if lowest_range != 0 else xs
```
Comment (luca-fiorito-11) [inline sandy/groupr.py:171]: @AitorBengoechea . This is ok, but pandas bfill could be better, i guess...

File: sandy/groupr.py [hunk 2]
```diff
@@ -197,15 +218,34 @@ def get_flux(self, **kwargs):
         (821000.0, 2231000.0]     1.41000e+06
         (2231000.0, 10000000.0]   7.76900e+06
         Name: iwt, dtype: float64
+
+        >>> endf6 = sandy.get_endf6_file('jeff_33','xs', 922350)
+        >>> groupr = endf6.get_gendf(ek=sandy.energy_grids.CASMO12)
+        >>> groupr.get_flux(mt=4)
```
Comment (luca-fiorito-11) [inline sandy/groupr.py:224]: Why does the flux change with the MT?

**Review by luca-fiorito-11 — CHANGES_REQUESTED** [review]

File: sandy/groupr.py (hunk 2 above)
Comment (AitorBengoechea) [reply sandy/groupr.py:224]: You are right, the flow should not change. It is a mistake to apply the same rule to flux and xs.

File: sandy/groupr.py (hunk 1 above)
Comment (AitorBengoechea) [reply sandy/groupr.py:171]: As it was: xs in the array of the values of the xs and columns is the `pd.MultiIndex` and index is the `pd.Index`. For me is more natural continue working with np in the xs variable and them in then combine all in the `data.append`

File: sandy/groupr.py (hunk 2 above)
Comment (luca-fiorito-11) [reply sandy/groupr.py]: ok, then should we fix the mistake or not?

**Review by luca-fiorito-11 — APPROVED** [review]

File: sandy/groupr.py (hunk 2 above)
Comment (luca-fiorito-11) [reply sandy/groupr.py]: i see it is fixed now

**Pull request merged**